infrastructure-scripts/claude_start.sh

```
@@ -0,0 +1,1640 @@
+
+                        echo -e "${BLUE}🔗 Connecting to existing instance $INSTANCE_ID at $SSH_HOST:$SSH_PORT${NC}"
+
+                        # Kill any existing tunnels on port 8000
+                        pkill -f "ssh.*8000" 2>/dev/null || true
+                        sleep 1
+
+                        # Create SSH tunnel (bypass host key verification for automation)
+                        # Clean up any existing tunnel first
+                        cleanup_ssh_tunnel
+                        ssh -N -L 8000:localhost:8000 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -p "$SSH_PORT" root@"$SSH_HOST" 2>/dev/null &
+                        local TUNNEL_PID=$!
```

Shellcheck (0.10.0)

[error] 'local' is only valid in functions.

(SC2168)